users/serializers.py

```diff
@@ -7,7 +7,23 @@ class Meta:
         fields = ['id', 'username', 'email', 'password', 'role']
         extra_kwargs = {'password': {'write_only': True}}
 
+
+    def create(self, validated_data):
+        role = validated_data.get('role')
+        if role and role.lower() != 'employee':
+            raise serializers.ValidationError("Role must be 'employee'.")
+        user = CustomUser.objects.create_user(**validated_data)
+        return user
+
+class ManagerSerializer(UserSerializer):
```

Comment:
Doesn't makes sense to create serializer for the Manager role.

You need to makes a command to create the manager role.